backend/kernelCI_app/management/commands/notifications.py

```diff
@@ -748,6 +790,11 @@ def add_arguments(self, parser):
             help="Limit checkout summary to specific origins (comma-separated list)."
             + " If not provided, any origin will be considered",
         )
+        parser.add_argument(
+            "--skip-sent-reports",
+            action="store_true",
+            help="Skip reports that have already been sent",
+        )
```

Comment:
Add a change to modify the management command invocation to include this option.